wweval/R/ms_fig2.R

```diff
@@ -135,8 +135,8 @@ make_fig2_ct <- function(ww_quantiles,
                          n_calib_days = 90,
                          max_n_site_labs_to_show = 3) {
   ww <- ww_quantiles |>
-    dplyr::filter(location == loc_to_plot) |>
-    dplyr::filter(forecast_date == date_to_plot) |>
+    dplyr::filter(location == !!loc_to_plot) |>
+    dplyr::filter(forecast_date == !!date_to_plot) |>
     dplyr::filter(
       date <= forecast_date + lubridate::days(n_forecast_days),
       date >= forecast_date - lubridate::days(n_calib_days)
```

Comment:
```suggestion
      date >= forecast_date - lubridate::days(!!n_calib_days)
```

And also below

```R
    dplyr::filter(lab_site_index <= max_n_site_labs_to_show)
```

should be

```R
    dplyr::filter(lab_site_index <= !!max_n_site_labs_to_show)
```